setup/l10n_it_fatturapa/setup.py

```diff
@@ -2,5 +2,11 @@
 
 setuptools.setup(
     setup_requires=['setuptools-odoo'],
-    odoo_addon=True,
+    odoo_addon={
+        'external_dependencies_override': {
+            'python': {
+                'pyxb': 'PyXB==1.2.6',
```

Comment:
```suggestion
                'pyxb': 'PyXB==1.2.5',
```
stesse considerazioni di https://github.com/OCA/l10n-italy/pull/1517#pullrequestreview-317639338